### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_Eclipse Select Subscriptions_Screen Reader: When checkbox is unchecked, Narrator is reading the list item as selected

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3327]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
Screen Reader: Narrator
MAS Violated: MAS1.3.1

**Note: Similar issue is found in the following Screens**
1.	Create Authentication Files

**Repro Steps:** 
1) Start Narrator and Open Java EE - Eclipse application
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Tab till tools-->Azure-->Azure Sign in and press enter
4) Select 'Device login' and tab till sign in and press enter
5) Tab till 'Copy and Open' button and press enter
6) Paste the code in the browser and open the Application
7) Verify When checkbox is unchecked, Narrator is reading the state of the list item properly

**Actual Result:** 
When checkbox is unchecked, Narrator is reading the list item as selected instead of reading as non-selected

**Expected Result:** 
When checkbox is unchecked, Narrator should read the list item as non-selected

**User Impact:** 
Screen Reader users will not know that the subscription is not selected if narrator does not read the status properly

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

![Bug_9_Create Authentication Files](https://user-images.githubusercontent.com/41109135/62204560-0c616600-b3ab-11e9-83df-7cee870115fe.png)
[Bug_9_When checkbox is unchecked narrator is reading selected.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451197/Bug_9_When.checkbox.is.unchecked.narrator.is.reading.selected.pptx)

**Comment by Flanker32** [comment]
This is the behavior of Narrator, which will read all items in Table as selected, we tested in a similar ui in Eclipse setting panel and it behaviors same.
![image](https://user-images.githubusercontent.com/12445236/64221204-96e33b00-cefe-11e9-8d66-946efc60552e.png)

And user could use NVDA to get the correct messages, here is a screenshot.
![image](https://user-images.githubusercontent.com/12445236/64221529-c777a480-ceff-11e9-9a90-fd5d03dcb790.png)

**Comment by Flanker32** [comment]
- Importance of experience: Medium, users may be confused when select subscriptions.
- Quality of experience: not impacted. Users could get the correct message by using NVDA.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
@Tanushagottipati Please have a review.

**Comment by soumyatippapuram** [comment]
@Flanker32 Please provide the link to your AAT. This is required to follow Exclusion process. For more details, please refer : https://microsoft.sharepoint.com/teams/ceacc/SitePages/External%20Bug%20Process.aspx

**Comment by Flanker32** [comment]
Linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), @soumyatippapuram please help us review it, thanks a lot!

**Comment by Tanushagottipati** [comment]
External Bug: https://microsoft.visualstudio.com/OS/_workitems/edit/20630112

**Comment by Tanushagottipati** [comment]
According to C&AI External Bug process verified with other free screen reader NVDA and the issue not present. As External narrator bug is logged closing this bug
![Screenshot (429)](https://user-images.githubusercontent.com/41109135/64427897-efb5fd80-d0cf-11e9-8d8e-443c0da6dd52.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749745
